activerecord/test/cases/adapters/postgresql/citext_test.rb

```diff
@@ -71,6 +71,12 @@ def test_select_case_insensitive
     assert_equal "Cased Text", x.cival
   end
 
+  def test_case_insensitiveness
+    attr = Citext.arel_table[:cival]
+    comparison = @connection.case_insensitive_comparison(attr, nil)
+    assert_no_match(/lower/i, comparison.to_sql)
```

Comment:
```suggestion
    assert_no_match(/LOWER/, comparison.to_sql)
```

Reply:
For an `assert_no_match`, IMO the case-insensitive version is safer -- we want to be sure no future version of a `lower()` call appears, regardless of case.